src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/run_conformance.py

```diff
@@ -88,13 +87,17 @@ def parse_arguments():
     ov_help = "OV binary files path. The default way is try to find installed OV by INTEL_OPENVINO_DIR in environmet variables or to find the absolute path of OV repo (by using script path)"
     working_dir_help = "Specify a working directory to save all artifacts, such as reports, models, conformance_irs, etc."
     type_help = "Specify conformance type: `OP` or `API`. The default value is `OP`"
+    nworkers_help = "Specify number of workers to run in parallel. Set to -1 to utilize all threads. The default value is 1"
+    gtest_filter_helper = "Specify gtest filter to apply when running test. E.g. *Add*:*BinaryConv*. The default value is None"
     dump_conformance_help = "Set '1' if you want to create Conformance IRs from custom/downloaded models. In other cases, set 0. The default value is '1'"
 
     parser.add_argument("-m", "--models_path", help=models_path_help, type=str, required=False, default=NO_MODEL_CONSTANT)
     parser.add_argument("-d", "--device", help= device_help, type=str, required=False, default="CPU")
     parser.add_argument("-ov", "--ov_path", help=ov_help, type=str, required=False, default=get_ov_path())
     parser.add_argument("-w", "--working_dir", help=working_dir_help, type=str, required=False, default=get_default_working_dir())
     parser.add_argument("-t", "--type", help=type_help, type=str, required=False, default="OP")
+    parser.add_argument("-nw", "--num_workers", help=nworkers_help, type=int, required=False, default=1)
```

Comment:
`-w`, `--workers`. Moreover Use the same as for gtest parallel. Default value should be `CPU count -1`